### PR TITLE
[backend/frontend] Add relations between entities

### LINKED ISSUE #3194: Add a new relationship "Domain-Name" belongs-to "Identity" (WHOIS)
## Use case

- [ ] **`MVP`** Add new STIX CORE relationships (new-sro): 
  - "Domain-Name" belongs-to "Identity" (WHOIS)
  - "IPv4" belongs-to "Identity"
  - "IPv6" belongs-to "Identity"

### PR DESCRIPTION
**Pull request by yassine-ouaamou**
### Proposed changes

Add Stix Core Relationships :
 - "Domain-Name" belongs-to "Identity" (Orgnization or Individual)
- "IPv4" belongs-to "Identity" (Orgnization or Individual)
- "IPv6" belongs-to "Identity" (Orgnization or Individual)

### Related issues

* #3194 

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by RomuDeuxfois** [comment]
If you have time, can you retrieve the available relationship by entity type through the API ? To avoid duplicate code and let the API give to the front all the structure relationship information.

And remove the relationsTypesMapping in front.

Of course, it's still a Nice-To-Have refacto

**Comment by yassine-ouaamou** [comment]
> If you have time, can you retrieve the available relationship by entity type through the API ? To avoid duplicate code and let the API give to the front all the structure relationship information.
> 
> And remove the relationsTypesMapping in front.
> 
> Of course, it's still a Nice-To-Have refacto

@RomuDeuxfois I will check how to do that

File: opencti-platform/opencti-graphql/src/database/stix.ts [hunk 1]
```diff
@@ -1,83 +1,9 @@
...
+import { ENTITY_AUTONOMOUS_SYSTEM, ENTITY_DOMAIN_NAME, ENTITY_HASHED_OBSERVABLE_ARTIFACT, ENTITY_HASHED_OBSERVABLE_STIX_FILE, ENTITY_HOSTNAME, ENTITY_IPV4_ADDR, ENTITY_IPV6_ADDR, ENTITY_MAC_ADDR, ENTITY_MEDIA_CONTENT, ENTITY_SOFTWARE, ENTITY_TEXT, ENTITY_URL, ENTITY_USER_ACCOUNT, isStixCyberObservable } from '../schema/stixCyberObservable';
+import { RELATION_ATTRIBUTED_TO, RELATION_AUTHORED_BY, RELATION_BASED_ON, RELATION_BEACONS_TO, RELATION_BELONGS_TO, RELATION_COMMUNICATES_WITH, RELATION_COMPROMISES, RELATION_CONSISTS_OF, RELATION_CONTROLS, RELATION_COOPERATES_WITH, RELATION_DELIVERS, RELATION_DERIVED_FROM, RELATION_DETECTS, RELATION_DOWNLOADS, RELATION_DROPS, RELATION_EXFILTRATES_TO, RELATION_EXPLOITS, RELATION_HAS, RELATION_HOSTS, RELATION_IMPERSONATES, RELATION_INDICATES, RELATION_INVESTIGATES, RELATION_LOCATED_AT, RELATION_MITIGATES, RELATION_ORIGINATES_FROM, RELATION_OWNS, RELATION_PART_OF, RELATION_PARTICIPATES_IN, RELATION_PUBLISHES, RELATION_RELATED_TO, RELATION_REMEDIATES, RELATION_RESOLVES_TO, RELATION_REVOKED_BY, RELATION_SUBTECHNIQUE_OF, RELATION_TARGETS, RELATION_USES, RELATION_VARIANT_OF } from '../schema/stixCoreRelationship';
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-graphql/src/database/stix.ts]: Why have the imports been formatted in one line ? Isn't it too long ?

Comment (yassine-ouaamou) [reply opencti-platform/opencti-graphql/src/database/stix.ts]: Indeed! My linter is not properly configured. I will fix this

**Comment by SouadHadjiat** [comment]
> If you have time, can you retrieve the available relationship by entity type through the API ? To avoid duplicate code and let the API give to the front all the structure relationship information.
> 
> And remove the relationsTypesMapping in front.
> 
> Of course, it's still a Nice-To-Have refacto

Do we have an open issue for this ? Maybe we could do it in a separate PR ? I like this small PR.

**Comment by yassine-ouaamou** [comment]
> > If you have time, can you retrieve the available relationship by entity type through the API ? To avoid duplicate code and let the API give to the front all the structure relationship information.
> > And remove the relationsTypesMapping in front.
> > Of course, it's still a Nice-To-Have refacto
> 
> Do we have an open issue for this ? Maybe we could do it in a separate PR ? I like this small PR.

@SouadHadjiat @RomuDeuxfois 
I will create a new branch for this and work on it separately

**Review by SouadHadjiat — APPROVED** [review]

**Pull request merged**